lib/wda/utils.js

```diff
@@ -56,7 +56,9 @@ async function checkForDependencies (bootstrapPath, useSsl = false) {
     let carthagePath = await fs.which('carthage');
     log.debug(`Carthage found: '${carthagePath}'`);
   } catch (err) {
-    log.errorAndThrow('Carthage not found. Install using `brew install carthage`');
+    log.errorAndThrow('Carthage binary is not found. Install using `brew install carthage` if it is not installed ' +
+      'and make sure the root folder, where carthage binary is installed, is present in PATH environment variable. ' +
+      'The current PATH value: ' + (process.env('PATH') ? `'${process.env('PATH')}'` : '<not defined for the Appium process>'));
```

Comment:
This is not how you access the environment. [process.env](https://nodejs.org/api/process.html#process_process_env) is an object, not a function. Also this should use string templates rather than concatenation.

Reply:
fixed